### ISSUE TITLE
The administrative division in France has been modified in the beginning of 2016 ?

### ISSUE THREAD

**Issue by A-contresens** [google/google-visualization-issues#2221]
Since 2016, the previously 27 administrative regions have been reduced to 18.
So the provinces display mode for France is wrong.

https://en.wikipedia.org/wiki/Regions_of_France
http://www.geonames.org/FR/administrative-division-.html

**Comment by carmoreira** [comment]
This is also part of ISO which the visualization API follows: https://www.iso.org/obp/ui/#iso:code:3166:FR (13 metro regions)

Can we expect an update from the available map or a way to load the new map with an option?

**Comment by loic-bellinger** [comment]
Yeah. GeoChart API feels SO OUTDATED. Not being able to display the correct regions and their subdivisions ("départements") is a real shame.

**Comment by amrsobhy** [comment]
This hasn't been solved yet?
